supertokens_python/querier.py

```diff
@@ -166,11 +166,18 @@ async def f(url: str) -> Response:
 
         return await self.__send_request_helper(path, "POST", f, len(self.__hosts))
 
-    async def send_delete_request(self, path: NormalisedURLPath):
+    async def send_delete_request(
+        self, path: NormalisedURLPath, params: Union[Dict[str, Any], None] = None
+    ):
+        if not params:
```

Comment:
I'm being pedantic here. But, use `params is None` instead. The reason is `not {}` also evaluates to `True`.